em_planner_ros/Obstacle[] obstacle_list
em_planner_ros/Obstacle[] obstacle_list_history

